samples/bluetooth/direction_finding_connectionless_rx/boards/nrf52833dk_nrf52820.conf

```diff
@@ -0,0 +1,10 @@
+CONFIG_BT_CTLR=y
+CONFIG_BT_LL_SW_SPLIT=y
+
+CONFIG_BT_CTLR_ADV_EXT=y
+CONFIG_BT_CTLR_ADV_PERIODIC=y
+
+# Enable Direction Finding TX Feature including AoA and AoD
+CONFIG_BT_CTLR_DF=y
+CONFIG_BT_CTLR_DF_ANT_SWITCH_RX=n
```

Comment:
`RX feature`?
The disabled options should related to TX not RX.

Reply:
The whole file content is wrong. I've changed it to proper one.